app/javascript/components/components/Footer.test.js

```diff
@@ -16,7 +16,9 @@ Enzyme.configure({ adapter: new Adapter() })
 describe("When Footer renders", () => {
   it("displays a heading", () => {
     const footer = shallow(<Footer />)
-    const footerHeading = footer.find("h3")
-    expect(footerHeading.text()).toEqual("This should fail")
+    const footerHeading = footer.find("NavItem")
```

Comment:
Hey - please check your tests.  What is line 20 testing?  This same comment for your Header test.  :)